### PR TITLE
Add getter for userActionsQueue

### PR DESCRIPTION
**Pull request by davidkwlam**
This will be useful down the road

### CODE REVIEW

File: src/game/game2048.ts [hunk 1]
```diff
@@ -42,6 +42,10 @@ export interface IGameState {
     return this.scores;
   }
 
+  public getUserActionsQueue() {
+    return this.userActionsQueue;
+  }
```
Comment (davidkwlam) [inline src/game/game2048.ts:47]: Just a follow up from https://github.com/TheUnblockedDemo/2048/pull/10. This feels like something we'll need eventually.

Comment (dennispiUB) [reply src/game/game2048.ts:47]: Great, let's go ahead and do this!

Comment (dennispiUB) [reply src/game/game2048.ts:47]: What's the delay? It's been 30 seconds! GO!

--

Comment posted using [Unblocked](https://getunblocked.com/dashboard/team/461c33ed-2d8d-468d-9409-d2cec26e4741/thread/46dc6a99-9f01-4cac-b09d-916f0ca18645?message=3f38b479-4aaa-4f99-a447-e753ffe72d87).

Comment (davidkwlam) [reply src/game/game2048.ts:47]: Will do!

--

Comment posted using [Unblocked](https://getunblocked.com/dashboard/team/461c33ed-2d8d-468d-9409-d2cec26e4741/thread/46dc6a99-9f01-4cac-b09d-916f0ca18645?message=871f978b-faae-46dd-8632-ddf1b9f42623).

**Pull request merged**